### PR TITLE
[release/3.1.4xx] Merging internal to public

### PR DESCRIPTION
**Pull request by joeloff**
Internal to public merge
Branding updated

### CODE REVIEW

**Comment by sfoslund** [comment]
@mmitche I don't recognize this leg but it seems to be failing repeatedly on both 3.1 PRs. Is it still in support and necessary?

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -42,13 +42,13 @@
       <Sha>dc3f4eef2b4e609adbc4daf0c7c67e6212db6ad8</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NET.Sdk.Razor" Version="3.1.11">
-      <Uri>https://github.com/dotnet/aspnetcore-tooling</Uri>
-      <Sha>2dda073cee874e290dc8558e4bea0786965aa9d8</Sha>
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore-tooling</Uri>
```
Comment (joeloff) [inline eng/Version.Details.xml:45]: Should these internal items be removed?

**Comment by sfoslund** [comment]
It looks like Jessie went out of support in June, I'm removing that leg: https://www.debian.org/releases/jessie/

File: eng/Version.Details.xml (hunk 1 above)
Comment (mmitche) [reply eng/Version.Details.xml:45]: Nope, they are valid.

**Review by mmitche — APPROVED** [review]

**Pull request merged**